docs/cluster-management.md

```diff
@@ -1 +1,43 @@
-// TODO
\ No newline at end of file
+# Multi-Cluster Management
+
+Multi-cluster management is a common task when it comes to managing your workloads on Amazon EKS. The management of these clusters adds a level of complexity especially when it comes to managing these clusters across multiple regions. Infrastructure as code (IaC) tools like the AWS CDK provides a seemless experience when it comes to the management of your clusters. 
```

Comment:
Multi-cluster management refers to the strategies associated with managing and updating cluster configuration across many Amazon EKS clusters.

Reply:
Do we want to include something on managing clusters to support hybrid workloads for when EKS-A comes in to play?